frontend/ui/data/keyboardlayouts/keypopup/en_popup.lua

```diff
@@ -43,26 +43,28 @@ return {
         north = "Ä",
         northeast = "Á",
         northwest = "À",
-        east = "Â",
-        west = "Ã",
+        east = "Â", -- A with circumflex (Romanian)
+        west = "Ă", -- A with breve (Romanian)
         south = "Ą",
         southeast = "Æ",
         southwest = "Å",
         "Ā",
         "Ǎ",
+        "Ã",
     },
     _a_ = {
         "a",
         north = "ä",
         northeast = "á",
         northwest = "à",
-        east = "â",
-        west = "ã",
```

Comment:
Ok, I will switch æ with ă and leave ã in its place, if this is ok.
I am also working on a separate Romanian keyboard.

Reply:
Well, as the keyboard layout shows, characters useful to Romanian were not a priority since that's considered as probably something better left to a Romanian keyboard, hence why they ended up in the extra row that isn't reachable by swipe without popup. The idea isn't that the US International style QWERTY layout should be ideal for everything without modifications, although it should at least be usable for a lot of languages written using the Latin alphabet. It's easier to hold for a popup and go for the extra row than to switch keyboards. When I wrote the above I forgot for a second that the æ is a common character in the Scandinavian languages. It's in English & French that it's rare. This simply betrays my own particular familiarity with Dutch, English, French, and German.

Romanian is unfortunately not part of the traditional US International type layout, see [below](https://github.com/koreader/koreader/pull/5652#discussion_r352285677). In that sense moving the Polish/Lithuanian ą would be a better idea if anything is to be moved within the a. That particular character is there mainly by analogy with a more traditional character like the `ç`.

Reply:
But note that as per the French & Spanish mods, a "Romanian keyboard" could literally just be the swap you're proposing here.

Reply:
As I will come with a PR for a separate Romanian keyboard, feel free to rearrange the Romanian ă with breve (which was missing, do not confuse with ã with tilde) whereever you consider the least impact here :)

Reply:
You mean caron, not tilde (`~`). But indeed, I didn't realize the difference at a glance. :-)

In light of that misinterpretation it would probably make the most sense right next to the a with caron. That's the space for characters that aren't strictly speaking part of the keyboard layout but more of a bonus feature. However, like I said the ą is also a kind of bonus feature. Since you say a Romanian keyboard is coming, I'd be more inclined to add it rather than changing any current positions.

Btw @robert00s how does the English QWERTY keyboard work for Polish atm?